alembic/versions/8c089a511ab7_create_plants.py

```diff
@@ -0,0 +1,35 @@
+"""create plants
+
+Revision ID: 8c089a511ab7
+Revises:
+Create Date: 2020-07-20 17:43:15.553458
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '8c089a511ab7'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+    'plants',
+        sa.Column('id', sa.Integer, primary_key=True),
+        sa.Column('name', sa.String(50)),
+        sa.Column('plant_type', sa.String(50)),
+        sa.Column('image', sa.Unicode(255)),
```

Comment:
We're assuming this means textfield, once we begin seeding the db we should be able to tell if this needs to be changed

Reply:
If it doesn't mean that, we'll get to practice more migrations!